Sources/Asset.swift

```diff
@@ -9,30 +9,43 @@
 import Foundation
 import ObjectMapper
 
+
+public extension String {
+
+    /**
+     Will make a `URL` from the current `String` instance if possible.
+     */
+    public func url() throws -> URL {
+        guard let url = URL(string: self) else { throw SDKError.invalidURL(string: self) }
+
+        return url
+    }
+}
+
```

Comment:
@dborsatto I think here is fine for now as the only use is to create a `URL` specifically for Contentful assets. The difference with my wrapper method compared to that provided by `Foundation` (the standard lib) is that I'm `throw`'ing and error here, whereas the regular initializer for `URL` will simply return nil. 

In other words, this convenience method is not intended for use with other url strings, and is meant to provide more debug info by returning the string that was unable to be transformed upon failure.

I'm planning on starting to arrange the project into subdirectories in the coming weeks and reorganizing code so maybe then I'll find a better place for it.